lib/Tooling/Refactor/RenameIndexedFile.cpp

```diff
@@ -333,12 +372,18 @@ static void findMatchingTextualOccurrences(
                                     Range, MatchHandler);
         CommentMatches.clear();
       }
-    } else if (!SelectorParsers.empty()) {
+    } else if (ScanNonCommentTokens) {
       for (auto &Parser : SelectorParsers) {
         if (Parser.handleToken(RawTok))
           MatchHandler(OldSymbolOccurrence::MatchingSelector,
                        Parser.SelectorLocations, Parser.SymbolIndex);
       }
+      for (auto &Parser : StringParsers) {
```

Comment:
Why is StringParser `checkToken` while the selector parser uses `handleToken`?

Reply:
Fixed